test/channels/player_channel_test.exs

```diff
@@ -1,62 +1,75 @@
 defmodule PlayerChannelTest do
   use Yggdrasil.ChannelCase
-  alias Yggdrasil.PlayerChannel
-  alias Yggdrasil.Message
+  alias Yggdrasil.{PlayerChannel, Message, User, Game, Character}
```

Comment:
Elixir 1.2 introduced this syntax for importing aliases, there were enough here that I went for it, but maybe we can make a pass to switch to this new syntax. I think it's a little cleaner when you have several imports.